### DISCUSSION TITLE
Docker and volumes - best practice

### DISCUSSION THREAD

**Discussion by sascha-schwegelbauer** [netbox-community/netbox-docker#1173] [Getting Help]
Hi,

I've just installed netbox using docker compose - works fine.
Now I'd like to mount the working data (i.e. netbox-postgres-data) on an external volume (/mnt/data/netbox).
All my attempts to put that into docker-compose.override.yml failed.
Changing the main docker-compose.yml feels not like the optimal way to go.
I'm happy about all hints / examples how to archive that.

Thanks in advance!

**Comment by aascedu** [comment]
Did you get a workaround ? I am looking to that as well as add docker volume to use as Local Datasource.

**Reply by aascedu** [reply]
If you want to specify the changes in your docker-compose.override.yml like so :
```version: '3.4'
services:
  netbox:
    container_name: netbox-v4.1.1-$NAME
    image: netbox:v4.1.1-plugins
volumes:
  netbox-postgres-data:
     driver:local
      etc...
```
      
It won't work since override seems to be only for the services part of the compose. At least I couldn't make it work like that.

What I did was change the original docker-compose.yml file with a script and since we decided to stick to a version of netbox-docker, it is viable until further problems 😉
